mantidproject/mantid
# Allow user to manually specify merge point

This work, adds the ability to specify the cross-over point between the LAB, merged data and the HAB in SANS reduction. It is currently implemented only for the new GUI. 

**To test:**
- Get the SANS data from http://download.mantidproject.org/. The ISIS sample data has some SANS data in a folder called loqdemo.
- Add this folder to your directories
- Open mantid, go to interfaces->SANS->ISIS SANS V2
- Load the maskfile.txt from the folder as a user file.
- Load the batch_mode_reduction.csv as the batch file
- Deletes the output name from the row
- Go to Settings-> General and select All reduction
- Go to runs and click Process, Wait
- Go to Settings->General and select merged reduction
- Tick Use Range Box, put 0.1 and 0.2 in the two boxes below.
- Go to runs and click Process, Wait
- Plot all three output workspaces
- Confirm that the merged reduction matches LAB below 0.1, HAB above 0.2 and is an average of the two between these two values.

Code review



Fixes #21027 . 

**Release Notes** 
Has been added to release notes.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
